opentelemetry-appender-tracing/src/layer.rs

```diff
@@ -513,7 +521,7 @@ mod tests {
             let span_id = cx.span().span_context().span_id();
 
             // logging is done inside span context.
-            log::error!("log from log crate");
+            log::error!(target: "my-system", "log from log crate");
```

Comment:
do we need these changes?

Reply:
Yes we can remove the target, will follow up in subsequent cleanups.